packages/pulpcore-wrappers/pulpcore-wrappers.spec

```diff
@@ -0,0 +1,42 @@
+%global wrappers gunicorn rq
+
+Name:           pulpcore-wrappers
+Version:        1.0.0
+Release:        1%{?dist}
+Summary:        pulpcore SELinux wrappers
+
+License:        GPL2+
+URL:            https://pulpproject.org
+Source0:        empty-file
+
+BuildArch:      noarch
+
+Requires:       python3-pulpcore
+
+%description
+pulpcore SELinux wrappers
+
+%prep
+# nothing
+
+%build
+for wrapper in %{wrappers}
+do
+  printf '#!/bin/bash\nexec %s "$@"\n"' ${wrapper} > ${wrapper}
```

Comment:
The number of `"` is 3. Is that correct?

Reply:
sure ;)